collectd-mlab.spec

```diff
@@ -77,17 +83,41 @@ install -D -m 644 viewer/collection-mlab.conf		%{buildroot}/etc/collection-mlab.
 install -D -m 644 plugin/collectd			%{buildroot}/etc/default/collectd
 install -D -m 644 plugin/collectd-mlab.conf		%{buildroot}/etc/collectd-mlab.conf
 install -D -m 644 plugin/types.db			%{buildroot}/usr/share/collectd-mlab/types.db
-install -D -m 644 plugin/mlab.py			%{buildroot}/var/lib/collectd/python/mlab.py
+install -D -m 644 plugin/mlab.py			%{buildroot}/var/lib/collectd/python/mlab_vs.py
+
+# Init script.
+install -D -m 755 plugin/collectd-setup		%{buildroot}/etc/init.d/collectd-setup
+
+# Python site-packages modules.
+install -d %{buildroot}/%{site_packages}/mlab/disco
+install -D -m 644 site-packages/mlab/disco/__init__.py \
+                  site-packages/mlab/disco/arp.py \
+				  site-packages/mlab/disco/collectd.py \
```

Comment:
No. All the mixed indents are due to my editor settings not being applied to `*.spec` files. I need to find a way to fix this.